documentation/library_guide/requirements.txt

```diff
@@ -1,17 +1,17 @@
-breathe==4.9.1
-sphinx==4.4.0
-jinja2==3.1.4
-# https://github.com/sphinx-doc/sphinx/issues/9923
-docutils==0.15
-# Pin versions to avoid a compatibility issue: "The sphinxcontrib.<extension_name> extension used by this project needs at least Sphinx v5.0"
-sphinxcontrib_applehelp<1.0.8
-sphinxcontrib_devhelp<1.0.5
-sphinxcontrib_htmlhelp<2.0.4
-sphinxcontrib_qthelp<1.0.6
-sphinxcontrib-serializinghtml<1.1.10
-
-sphinx_book_theme
-sphinx-intl==2.0.0
-sphinx-tabs
-sphinx-prompt
-sphinx_substitution_extensions
+breathe==4.9.1
+sphinx==4.4.0
+jinja2==3.1.4
+# https://github.com/sphinx-doc/sphinx/issues/9923
+docutils==0.15
+# Pin versions to avoid a compatibility issue: "The sphinxcontrib.<extension_name> extension used by this project needs at least Sphinx v5.0"
+sphinxcontrib_applehelp<1.0.8
+sphinxcontrib_devhelp<1.0.5
+sphinxcontrib_htmlhelp<2.0.4
+sphinxcontrib_qthelp<1.0.6
+sphinxcontrib-serializinghtml<1.1.10
+
+sphinx_book_theme==1.1.2
```

Comment:
Looking at another PR, it looks like this is correct:
sphinx_book_theme==1.0.1
sphinx-tabs==3.4.5
sphinx-prompt==1.5.0
sphinx_substitution_extensions==2022.2.16

Reply:
Thanks.  The version numbers I selected had been pulled from a previous PR, but I'm not able to find that PR now.

Reply:
I think it initially tries the numbers you originally had, but then backs them off later in the script due to dependencies.